addons/mail/static/src/components/chat_window_manager/tests/chat_window_manager_tests.js

```diff
@@ -1993,6 +1993,33 @@ QUnit.test('chat window with a thread: keep scroll position in message list on t
     );
 });
 
+QUnit.test('chat window with a thread: should not be able to reply on temporary/transient messages', async function (assert) {
+    assert.expect(1);
+
+    this.data['mail.channel'].records.push({
+        id: 20,
+        is_minimized: true,
+    });
+    await this.start();
+    await afterNextRender(() => {
+        document.querySelector(`.o_ComposerTextInput_textarea`).focus();
+        document.execCommand('insertText', false, "/");
+    });
+    await afterNextRender(() =>
+        document.querySelector('.o_ComposerSuggestion').click()
+    );
+    await afterNextRender(() => {
+        const kevt = new window.KeyboardEvent('keydown', { key: "Enter" });
+        document.querySelector('.o_ComposerTextInput_textarea').dispatchEvent(kevt);
+    });
+    await afterNextRender(() => document.querySelector('.o_Message').click());
+    assert.containsNone(
+        document.body,
+        '.o_MessageActionList_actionReply',
+        "should not have action to reply to temporary/transient messages"
+    );
+});
+
```

Comment:
I would just drop this test with chat window and just keep the test in discuss app.
It's duplicate with discuss with slight feature differences that are quite unrelated to the test at hand.